public/content/lessons/2016/cross-products/index.mdx

```diff
@@ -1,10 +1,253 @@
+/>
+
+To compute the cross product calculate the determinant of the transformation matrix $\left[\begin{array}{cc} 1 & 4 \\ -2 & 3\end{array}\right]$.
+
+<Figure
+    image="./figures/determinant-example/DeterminantExampleQuestion1Calculation.svg" 
+/>
+
+</Question>
+
+## Properties
+
+For example, you might notice that when two vectors are perpendicular, or close to being perpendicular, their cross product is larger than it would be if they were pointing in a very similar direction, because the area of that parallelogram is larger when its sides are perpendicular. 
+
+<Figure
+    image="./figures/properties/BiggerSmaller.svg"
+    caption="$\vec{\mathbf{v}} \times \vec{\mathbf{w}}$ is bigger when the vectors are more perpendicular to eachother and smaller when the vectors are pointing in a similar direction."
+/>
+
+Something else you might notice is that if you scale up one of the vectors, perhaps multiplying $\vec{\mathbf{v}}$ by $3$, then the area of that parallelogram is scaled up by a factor of three, so $(3\vec{\mathbf{w}})$ cross $\vec{\mathbf{w}}$ will be exactly three times the value of $\vec{\mathbf{v}}$ cross $\vec{\mathbf{w}}$.
+
+<Figure
+    image="./figures/properties/Scaling.svg" 
+/>
+
+## 3d cross product, standard view
+
+Now, even though everything we just did is a perfectly fine mathematical operation, what I just described is technically not the cross product. The true cross product is something that combines two three-dimensional vectors to get a new 3d vector. You still consider the parallelogram defined by the two vectors, and the area of that parallelogram still plays a big role. To be concrete let's say that area is 2.5 for the vectors shown here.
+
+<Figure
+    image="figures/3d-area-of-parallelogram.png" 
+/>
+
+But as I said, the cross product is not a number, it's a vector. This new vector's length will be the area of that parallelogram, which in this example is 2.5, and the direction of that new vector is perpendicular to that parallelogram.
+
+<Figure
+    image="figures/3d-resulting-vector.png" 
+/>
+
+<!--TODO: lock orientation of the axis so that the two views are identical-->
+
+But which way, right?  There are two possible vectors with length 2.5 pointing perpendicular to a given plane. This is where the right hand rule comes in. Point the forefinger of your right hand in the direction of v, stick out your middle finger in the direction of w; then when you point up your thumb, that's the direction of the cross product.
+
+<Figure
+    image="figures/3d-righthand-rule.png" 
+/>
+
+For example, let v be the vector with length 2 in the z direction, and let w be the vector with length 2 in the y direction. The parallelogram they define in this simple example is actually a square, since they're perpendicular, and the area of that square is 4. Using the right hand rule, their cross product should point in the negative x direction, so this cross product is -4*(i hat).
```

Comment:
```suggestion
For example, let v be the vector with length 2 in the z direction, and let w be the vector with length 2 in the y direction. The parallelogram they define in this simple example is actually a square, since they're perpendicular, and the area of that square is 4. Using the right hand rule, their cross product should point in the negative x direction, so this cross product is $-4 \hat{\imath}$.
```